src/main/groovy/jenkins/automation/builders/MultibranchPipelineJobBuilder.groovy

```diff
@@ -0,0 +1,60 @@
+package jenkins.automation.builders
+import jenkins.automation.utils.CommonUtils
+
+import javaposse.jobdsl.dsl.DslFactory
+import javaposse.jobdsl.dsl.jobs.MultibranchWorkflowJob
+
+
+/**
+ * Multibranch Pipeline Job builder creates a set of Pipeline projects according to detected branches in one SCM repository
+ * @param name Sets the job name
+ * @param description Sets a description for the folder
+ * @param branchSourceGithub if true, provide  ghOrganizationName, ghRepositoryName, ghScanCredentialsId and ghEndpoint
+ * @param ghOrganizationName Sets the name of the GitHub Organization or GitHub User Account
+ * @param ghRepositoryName Sets the name of the GitHub repository
+ * @param ghScanCredentialsId Sets scan credentials for authentication with GitHub
+ * @param ghEndpoint Sets the GitHub API URI
+ * @param branchSourceGit if true, provide ghRemote and ghCredentialsId
+ * @param ghRemote Sets the Git remote repository URL
+ * @param ghCredentialsId Sets credentials for authentication with the remote repository 
```

Comment:
I think we're missing a few params here, like `discardOldItems` and `oldNumToKeep`.